### PR TITLE
UI refactor

### PR DESCRIPTION
**Pull request by captainPrime**
### Worked On
- Removed unused module 
- Removed unused space

### CODE REVIEW

File: src/pages/SearchPage.js [hunk 1]
```diff
@@ -4,11 +4,6 @@ import { FaTwitter } from "react-icons/fa";
...
 import "../assets/css/search.css";
 import "../assets/css/searchPage.css";
-import {
-  getDocument,
-  createDocument,
-  deleteDocument,
-} from "../services/firebase-config";
 import Bookmarks from "../components/bookmarks";
 import Search from "../components/search";
 import Logo from "../components/logo";
```
Comment (emeka-osuagwu) [inline src/pages/SearchPage.js:9]: do we really need this logo here @captainPrime

File: src/pages/SearchPage.js [hunk 2]
```diff
@@ -4,11 +4,6 @@ import { FaTwitter } from "react-icons/fa";
 import "../assets/css/bookmark.css";
 import "../assets/css/search.css";
 import "../assets/css/searchPage.css";
-import {
```
Comment (emeka-osuagwu) [inline src/pages/SearchPage.js:7]: so where will the firebase config be

Comment (emeka-osuagwu) [reply src/pages/SearchPage.js:7]: so where will the firebase config be @captainPrime

**Review by emeka-osuagwu — CHANGES_REQUESTED** [review]

File: src/pages/SearchPage.js (hunk 2 above)
Comment (captainPrime) [reply src/pages/SearchPage.js:7]: fixed

**Pull request merged**